doc/man/hirte.conf.5.md

```diff
@@ -64,6 +66,20 @@ LogTarget=journald
 LogIsQuiet=false
 ```
 
+Using a value that is continued on multiple lines:
+
+```
+[hirte]
+ManagerPort=842
+AllowedNodeNames=agent-007,
+   agent-123,
+   agent-456,
+   agent-789,
```

Comment:
Hmm, wouldn't the `,` after the last node causing parsing error?

Reply:
No. The `,` is independent of the ini parsing. And later on when splitting the names by the `,` symbol we check if the node name is empty or NULL. The `cfg_dump` is a bit misleading (it just takes the read, full string), but via `list_nodes()` API call on the manager this can be verified.

Reply:
I agree. Removed it.